### ISSUE TITLE
 ios  Live activity

### ISSUE THREAD

**Issue by woxiqingxian** [sideshow/apns2#225]
Hi, seeing that there are two ios  Live activity merge requests. I wonder if the author is available to merge them?

https://github.com/sideshow/apns2/pull/219
https://github.com/sideshow/apns2/pull/223

**Comment by icep87** [comment]
@sideshow Will you be able to approve those to mentioned PR's? Or is there anyone else that can support with doing that?

**Comment by gregorosaurus** [comment]
Support for live activities is something that I think the community would benefit from.  There are multiple PR's, so I think implementing this or approving a PR should be the focus. :)

**Comment by genesor** [comment]
Hello @sideshow 

Is there any chance for you to have a look and maybe merge one of the two PRs (#223 seems a bit more advanced) so we can have LiveActivities supported directly in the library ? 

Cheers

**Comment by gondo** [comment]
This was implemented in #802 
This issue should be closed.
ping @GonzaloAvilez

**Comment by GonzaloAvilez** [comment]
you are right @gondo , thanks. i make a reference to this issue in another library , the changes are done there too. 
I don't have permissions to close this issue. Maybe @woxiqingxian could you help us to close this topic please
